### ISSUE TITLE
Feature Request: Add water tracker

### ISSUE THREAD

**Issue by HealerLFG** [simonoppowa/OpenNutriTracker#32]
Title says it all.  A way to track how many units of plain water you've drank would fit nicely here.  For a bonus to that feature, you can opt to select drink mixes that you may have added in as well

**Comment by encrustace** [comment]
Water reminder will be a great enhancement

**Comment by juliansko** [comment]
Hi, I am trying to implement this. Should the water tracker have the same green tone as the rest of the dashboard or should it be in blue to differentiate? @simonoppowa

**Comment by simonoppowa** [comment]
> Hi, I am trying to implement this. Should the water tracker have the same green tone as the rest of the dashboard or should it be in blue to differentiate? @simonoppowa 

I would keep it in sync with the current (green) color scheme to make it possible to use material you in the future

**Comment by Elxas866** [comment]
great idea!

**Comment by jordan-lee-code** [comment]
Hi @HealerLFG — confirming this is still outstanding on `develop`.

Code location: no water tracker. Grep for `water` / `hydration` finds only physical-activity entries (water aerobics) and unrelated l10n strings. Adding this means a new `WaterIntakeDBO` (next free Hive `typeId`), a quick-add UI on the home dashboard (likely small "+250ml" buttons), and a daily total surface alongside kcal / macros. Past maintainer guidance: keep visual styling green-themed in line with the existing dashboard.

Labelled `enhancement`, `help wanted`.

**Comment by jordan-lee-code** [comment]
PR up at #418.

Small water-drop chip next to the weight chip on home, slider dialog to log a sip, Undo last for mis-taps. Goal lives on Profile, defaults gendered from the EFSA 2010 AI minus food moisture (1500 ml women, 1900 ml men). Cited inline so the "2 litres of plain water" myth doesn't sneak back in.

Drink mixes deliberately out of scope for v1, happy to follow up once this lands.
